src/containers/blocks.jsx

```diff
@@ -72,17 +74,23 @@ class Blocks extends React.Component {
         addFunctionListener(this.workspace, 'zoom', this.onWorkspaceMetricsChange);
 
         this.attachVM();
+        this.props.vm.setLocale(this.props.locale, this.props.messages);
```

Comment:
This is a duplicate of line 63? I think you only want one of them?

Reply:
@paulkaplan earlier line is gone now :)